java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture2/tasks/_1/Number1.java

```diff
@@ -0,0 +1,167 @@
+            char operation = scanner.next().charAt(0);
+            System.out.println("1. Input second double:\n2. Get current value");
+
+            if (scanner.nextInt() == 2) {
+                System.out.println(numberOneDouble);
+            }
+            System.out.println("Input second double");
+
+           double numberTwoDouble = scanner.nextDouble();
+
+                switch (operation) {
+                    case '+' -> calculator.addition(numberOneDouble, numberTwoDouble);
+                    case '-' -> calculator.subtraction(numberOneDouble, numberTwoDouble);
+                    case '*' -> calculator.multiplication(numberOneDouble, numberTwoDouble);
+                    case '/' -> calculator.division(numberOneDouble, numberTwoDouble);
+                    default -> System.out.println("ERROR");
+                }
+            }
+        }
+    }
+
+
+class Calculator {
+
+    private boolean isInteger;
+
+    public void setInteger(boolean integer) {
+        isInteger = integer;
+    }
+
+    public void addition(double numberOneDouble, double numberTwoDouble) {
+        if (isInteger) {
+
+            int numberOneInt = (int) numberOneDouble;
+            int numberTwoInt = (int) numberTwoDouble;
+
+            int calculationForInteger = numberOneInt + numberTwoInt;
+            System.out.println("Result:" + " " + calculationForInteger);
+        }
+        else {
+            double calculationForDouble = numberOneDouble + numberTwoDouble;
+            System.out.println("Result:" + " " + calculationForDouble);
+        }
+    }
+
+    public void subtraction(double numberOneDouble, double numberTwoDouble) {
+        if (isInteger) {
+            int numberOneInt = (int) numberOneDouble;
+            int numberTwoInt = (int) numberTwoDouble;
+
+           int calculationForInteger = numberOneInt - numberTwoInt;
+            System.out.println("Result:" + " " + calculationForInteger);
+        }
+        else {
+           double calculationForDouble = numberOneDouble - numberTwoDouble;
+            System.out.println("Result:" + " " + calculationForDouble);
+        }
+    }
+
+    public void multiplication(double numberOneDouble, double numberTwoDouble) {
+        if (isInteger) {
+            int numberOneInt = (int) numberOneDouble;
+            int numberTwoInt = (int) numberTwoDouble;
+
+            int calculationForInteger = numberOneInt * numberTwoInt;
+            System.out.println("Result:" + " " + calculationForInteger);
+        }
+        else {
+            double calculationForDouble = numberOneDouble * numberTwoDouble;
+            System.out.println("Result:" + " " + calculationForDouble);
+        }
+    }
+
+    public void division(double numberOneDouble, double numberTwoDouble) {
+        int numberOneInt = (int) numberOneDouble;
+        int numberTwoInt = (int) numberTwoDouble;
+
+        if (!isInteger) {
+            if (numberOneDouble == 0.0 && numberTwoDouble == 0.0) {
```

Comment:
try to rewrite the if condition to only print a message when `numberTwo` is zero